be/src/util/mem_info.cpp

```diff
@@ -85,9 +86,9 @@ void MemInfo::refresh_allocator_mem() {
     // https://jemalloc.net/jemalloc.3.html
     // https://www.bookstack.cn/read/aliyun-rds-core/4a0cdf677f62feb3.md
     _s_allocator_cache_mem.store(get_je_all_arena_metrics("tcache_bytes") +
-                                         get_je_metrics("stats.metadata") +
                                          get_je_all_arena_metrics("pdirty") * get_page_size(),
                                  std::memory_order_relaxed);
+    _s_allocator_metadata_mem.store(get_je_metrics("stats.metadata"), std::memory_order_relaxed);
```

Comment:
add comment here to explain why not count metadata cache to allocator cache mem size.

Reply:
done